demo/test/dummy.test.js

```diff
@@ -28,11 +24,8 @@ describe('My super cool test', () => {
 		await assert.cookie('name').doesNot().exist();
 
 		// test element
-		await assert.element({css: 'div'}).matches({
-			name: PROP.LEFT,
-			type: COMP.LESSER,
-			val: 400,
-		});
+		await assert.element({css: 'div',
```

Comment:
This is not readable. See our style guide. There is eslint config in the project. Should be:
```
await assert.element({
    css: 'div',
    index: 1,
});
```

Reply:
will fix it, but I guess it is ok with current eslint config, as I understand according to docs https://eslint.org/docs/rules/object-curly-newline we need to remove `"consistent": true` from `"object-curly-newline": ["warn", { "consistent": true }]`